### PR TITLE
Fixes#192(Partly): added examples in all entertainment skills

### PR DESCRIPTION
**Pull request by madhavrathi**
Fixes issue #192 

Changes: Added examples to all entertainment skills 
@uday96 @rishiraj824 @DravitLochan @dynamitechetan Please review
Thanks

### CODE REVIEW

**Review by rishiraj824 — APPROVED** [review]

File: models/general/entertainment/en/cooking_guide.txt [hunk 1]
```diff
@@ -1,13 +1,15 @@
 #Gives recipies and links to cook a dish
 * cook *
 !console:To cook  $title$ , make sure you have $ingredients$ and check out $href$
+!example:how cook biryani
```
Comment (Manveer695) [inline models/general/entertainment/en/cooking_guide.txt]: Writing the !example after !console doesn't let the skill work. You can test it using [this](http://dream.susi.ai/p/coo) example.

File: models/general/entertainment/en/cooking_guide.txt [hunk 2]
```diff
@@ -18,6 +20,7 @@ eol
 #Ex: recipe list for biryani
 recipe list for *
 !console:
+!example:recipe list for biryani
```
Comment (Manveer695) [inline models/general/entertainment/en/cooking_guide.txt]: here too, shift the !example string somewhere above.

File: models/general/entertainment/en/cricket.txt [hunk 3]
```diff
@@ -1,5 +1,6 @@
 matches going on|current matches|Do you know about matches going on?|Do you know cricket matches going on?|Tell me about matches going on?|Tell me about current cricket matches|Do you know about matches going on|Do you know cricket matches going on|is there any cricket match today?|what are the cricket matches going on?|is there any cricket match today|what are the cricket matches going on|find current matches into play|any cricket match to know about?|what is going on in the world of cricket?|
 !console:
+!example:Do you know cricket matches going on?
```
Comment (Manveer695) [inline models/general/entertainment/en/cricket.txt]: here too.

**Review by Manveer695 — CHANGES_REQUESTED** [review]

**Review by uday96 — CHANGES_REQUESTED** [review]
!example must be written just after the query and before the response.

File: models/general/entertainment/en/cooking_guide.txt (hunk 1 above)
Comment (madhavrathi) [reply models/general/entertainment/en/cooking_guide.txt]: Thanks or pointing it out. I will fix this asap.
Thanks

**Comment by madhavrathi** [comment]
@Manveer695 @uday96 i have made the required changes. Please review.
Thanks

**Review by Manveer695 — APPROVED** [review]

**Pull request merged**